types/jasmine/index.d.ts

```diff
@@ -18,6 +18,7 @@
 //                 kirjs <https://github.com/kirjs>
 //                 Md. Enzam Hossain <https://github.com/ienzam>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.1
```

Comment:
Please revert this and clean up your local environment instead. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50675#discussion_r559388699 for a similar discussion.

Reply:
Reverted, thanks for the tip